src/Component/src/Metadata/Extractor/PhpFileResourceExtractor.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Metadata\Extractor;
+
+use Sylius\Resource\Metadata\ResourceMetadata;
+
+final class PhpFileResourceExtractor extends AbstractResourceExtractor
+{
+    /**
+     * @inheritdoc
+     */
+    protected function extractPath(string $path): void
```

Comment:
It's a resource extractor, not path extractor, so I guess `extractFromPath` is more suited name.

```suggestion
    protected function extractFromPath(string $path): void
```

Reply:
yes, indeed, that's a better name.